### PR TITLE
Add English starboard documentation

### PR DESCRIPTION
**Pull request by DEVTomatoCake**
## Type of PR

- [ ] Translation update / creation
- [x] Original file creation
- [ ] Original file content (not typo) update
- [x] Original file typo update

## Checks

If you have changed the content (/ meaning) of an original file, please confirm:

- [x] Yes, I have copied the updated section to all translations of this file manually.

## Notes

This also changes some of the generator files as well as the README, mostly to fix typos.
It also adds a log message which displays the location a file created using `node generate-docs-file.js`.

### CODE REVIEW

File: bin/generate-docs-file.js [hunk 1]
```diff
@@ -45,22 +45,22 @@ Explain what users can configure and include a [GLink](https://scnx.app/user/too
...
+## Troubleshooting {#troubleshooting}
+Think of possible issues users might run into (or search our public support channel for inspiration) and explain how to avoid / fix them.
 You can layout this section as you want - feel free to "peak" in other documentation files for inspiration.
 ${moduleData.hasDB ? `
 ## Stored data {#data-usage}
-As part of our transparency commitment, please explain what data is stored, when it is stored and how users can delete it. Need inspiration? 
-Look at other docs-files.
-` : ''}
+As part of our transparency commitment, please explain what data is stored, when it is stored and how users can delete it. Need inspiration?
+Look at other docs-files.` : ''}
```
Comment (DEVTomatoCake) [inline bin/generate-docs-file.js:60]: This is to remove the double empty line generated at the end if the module has a database.

File: bin/generate-docs-file.js [hunk 2]
```diff
@@ -45,22 +45,22 @@ Explain what users can configure and include a [GLink](https://scnx.app/user/too
...
+Think of possible issues users might run into (or search our public support channel for inspiration) and explain how to avoid / fix them.
 You can layout this section as you want - feel free to "peak" in other documentation files for inspiration.
 ${moduleData.hasDB ? `
 ## Stored data {#data-usage}
-As part of our transparency commitment, please explain what data is stored, when it is stored and how users can delete it. Need inspiration? 
+As part of our transparency commitment, please explain what data is stored, when it is stored and how users can delete it. Need inspiration?
 Look at other docs-files.
-` : ''}
-`;
+` : ''}`;
```
Comment (DEVTomatoCake) [inline bin/generate-docs-file.js:61]: This removes the double empty lines generated at the end of files.

**Review by SCDerox — APPROVED** [review]
Thanks for your contribution!

**Comment by SCDerox** [comment]
Thanks for your contribution - you have received a payout as part of our Open Source Developer Program. You can find details in your SCNX organisation payout history. To pay out the received amount, please follow instructions in your dashboard. If you have any questions, please contact ScootKit staff on scnx.app/help.
Cheers ✌️

**Pull request merged**